src/vald/v1/vald/flush_pb2.py

```
@@ -0,0 +1,41 @@
+    5,
+    28,
+    3,
+    '',
+    'vald/v1/vald/flush.proto'
+)
+# @@protoc_insertion_point(imports)
+
+_sym_db = _symbol_database.Default()
+
+
+from google.api import annotations_pb2 as google_dot_api_dot_annotations__pb2
```

Ruff

`google.api.annotations_pb2` imported but unused

Remove unused import: `google.api.annotations_pb2`

(F401)